### ISSUE TITLE
SyntaxError: export 'Html' not found in '@kitajs/html' in the latest version

### ISSUE THREAD

**Issue by Sketioo** [kitajs/html#261]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Versions

in your latest version 4.2.0

### Description

So right now I use Elysia.js to develop my application, and then I install @elysiajs/html and use it as documentation standard. After I wrote the code, it was supposed to work perfectly, but when I see the console, there is an error like this:

```ts
1 | (function (entry, fetcher)
    ^
SyntaxError: export 'Html' not found in '@kitajs/html'
```

This error appears in the latest version of @kitajs/html only. When I downgraded it to 4.1.0, it worked completely fine.

### Steps to Reproduce

```tsx
import { Elysia } from "elysia";
import { staticPlugin } from "@elysiajs/static";
import { html } from "@elysiajs/html";

const app = new Elysia()
  .use(html())
  .use(
    staticPlugin({
      prefix: "/",
    })
  )
  .get("/", () =>
    Bun.file(
      "/c/Users/Martio Husein Samsu/Documents/Perkuliahan/Semester 7/Workspace/BUN/crud-app/views/home.html"
    )
  )

  .listen(3000);

console.log(
  `🦊 Elysia is running at ${app.server?.hostname}:${app.server?.port}`
);
```

### Expected Behavior

No error

**Comment by arthurfiorette** [comment]
Hi @Sketioo i just released [4.2.1](https://github.com/kitajs/html/releases/tag/%40kitajs%2Fhtml%404.2.1) to fix this issue, can you confirm?

**Comment by Sketioo** [comment]
Its worked now, thanks!
